addons/sys_external/fnc_getSharedExternalRadios.sqf

```diff
@@ -20,6 +20,10 @@ params ["_unit"];
 private _radios = [_unit] call EFUNC(sys_core,getGear);
 private _radioList = _radios select {_x call EFUNC(sys_radio,isUniqueRadio)};
 
-if (!(alive _unit) || {captive _unit}) exitWith {_radioList};
+if (
+    !(alive _unit) ||
+    {captive _unit} ||
+    {_unit getVariable ["ACE_isUnconscious", false]}
```

Comment:
This might work both for ACE and any other medical system:
```suggestion
    {lifeState _unit isEqualTo "INCAPACITATED"}
```

Reply:
Neat!